### PR TITLE
Use multi-active-gw dependencies

### PR DESCRIPTION
**Pull request by skitt**
This ensures that GHAs, base branches, and images are all taken from
the multi-active-gw branches.

Signed-off-by: Stephen Kitt <skitt@redhat.com>

### CODE REVIEW

**Review by Billy99 — APPROVED** [review]
Thanks Stephen. This is a lot more work than I thought it would be to add a feature branch. We appreciate it.

**Comment by skitt** [comment]
> Thanks Stephen. This is a lot more work than I thought it would be to add a feature branch. We appreciate it.

You’re welcome! Our build setup is a bit too complex for its own good ;-). Next step: document all this, and perhaps improve some parts of it...

File: api/submariner/v1alpha1/zz_generated.deepcopy.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```
Comment (dfarrell07) [inline api/submariner/v1alpha1/zz_generated.deepcopy.go]: This is intentional?

**Review by dfarrell07 — APPROVED** [review]

File: api/submariner/v1alpha1/zz_generated.deepcopy.go (hunk 1 above)
Comment (skitt) [reply api/submariner/v1alpha1/zz_generated.deepcopy.go]: Ah no, it gets added automatically when building with Go 1.17. I’ve removed it.

**Pull request merged**